Lib/argparse.py

```diff
@@ -875,7 +875,7 @@ def __init__(self,
                 _option_strings.append(option_string)
 
         if help is not None and default is not None:
-            help += f" (default: {default})"
+            help += " (default: %(default)s)"
```

Comment:
@pablogsal, this works since the help string is treated as a formatting string and that way default display isn't duplicated.